bluepyemodel/evaluation/modifiers.py

```diff
@@ -370,6 +377,13 @@ def isolate_axon(sim=None, icell=None):
 
 def replace_axon_legacy(sim=None, icell=None):
     """Replace axon used in legacy thalamus project"""
+    if len([sec for sec in icell.axonal]) < 2:
+        raise ValueError(
+            "Less than two axon sections are present! "
+            "The replace_axon_legacy morph modifier cannot be applied to such a morphology. "
+            "Please use 'bluepyopt_replace_axon' instead to use "
+            "bluepyopt.ephys.morphologies.NrnFileMorphology default morph modifier."
```

Comment:
Can we have something similar to:
```suggestion
            "Please use 'bluepyopt_replace_axon'  or other valid modifiers from  "
            "https://github.com/BlueBrain/BluePyEModel/blob/main/bluepyemodel/evaluation/modifiers.py"
            "or a custom modifier"
```
?

Reply:
done in latest commit!